graylog2-web-interface/src/logic/pipelines/SourceGenerator.js

```diff
@@ -19,7 +19,7 @@ const SourceGenerator = {
     let source = `pipeline "${pipeline.title}"\n`;
 
     pipeline.stages.forEach((stage) => {
-      source += `stage ${stage.stage} match ${stage.match.toLowerCase()}\n`;
+      source += `stage ${stage.stage} match ${stage.match?.toLowerCase() || 'all'}\n`;
```

Comment:
The previous default was `either` so I think we should use that instead of `all`:

```suggestion
      source += `stage ${stage.stage} match ${stage.match?.toLowerCase() || 'either'}\n`;
```